scripts/.python-version

```diff
@@ -0,0 +1 @@
+<3.12
```

Comment:
What's the reason for this?

Reply:
It's to match what we had in the `pyproject.toml` but I'm not sure if it even works. I'll removee